src/api/todos.ts

```diff
@@ -6,3 +6,25 @@ export const getTodos = (userId: number) => {
 };
 
 // Add more methods here
+type Data = {
+  title?: string,
+  userId?: number,
+  completed: boolean,
+};
+
+type ObjData = {
+  id: number,
+  userId: number,
+  completed: boolean,
+  title: string,
+  createdAt: string,
+  updatedAt: string,
+};
+
+export const postTodos = (userId: number, data: Data): Promise<ObjData> => {
+  return client.post(`/todos?userId=${userId}`, data);
+};
+
+export const deleteTodos = (userId: number, todoId: number) => {
```

Comment:
```suggestion
export const deleteTodo = (userId: number, todoId: number) => {
```
You're deleting single todo, aren't you?